packages/lisk-transactions/src/0_transfer.ts

```diff
@@ -69,9 +58,8 @@ const validateInputs = ({
 	}
 
 	if (
-		recipientId &&
 		recipientPublicKey &&
```

Comment:
here recipientId is not inserted yet, so probably need to be checked?

Reply:
Fixed. Resolving.